src/dialogbox.h

```diff
@@ -23,7 +23,7 @@
 #ifndef DIALOGBOX_H_
 #define DIALOGBOX_H_
 
-#include <QtGui>
+#include <QtWidgets>
```

Comment:
Also the next code snippet ensures this builds for both versions of qt:
```
#include <QtGlobal>
#if QT_VERSION >= 0x050000
    #include <QtWidgets>
#else
    #include <QtGui>
#endif
```
Please note `QtGlobal` defines the `QT_VERSION`.